t/ui/18-tests-details.t

```diff
@@ -311,6 +312,63 @@ subtest 'running job' => sub {
     # waiting for the periodic call anyways
     $driver->execute_script('window.enableStatusUpdates = false');
 
+    subtest 'liveview loading animation is displayed' => sub {
+        # ensure we are on the Liveview tab
+        like(current_tab, qr/live/i, 'live tab active by default');
+
+        # switch to another tab (e.g., Details)
+        $driver->find_element_by_link_text('Details')->click();
+        like(current_tab, qr/details/i, 'switched to details tab');
+
+        # switch back to Live View tab
+        $driver->find_element_by_link_text('Live View')->click();
+        like(current_tab, qr/live/i, 'switched back to live tab');
+
+        # check that the liveview-loading div exists after tab switch
+        my $loading_element = $driver->find_element('#liveview-loading', 'css');
+        ok($loading_element, 'liveview-loading element exists after tab switch');
+
+        # check that the liveview-loading div is visible after tab switch
+        ok($loading_element->is_displayed(), 'liveview-loading is visible after tab switch');
+    };
+
+    subtest 'liveview loading animation hides after live view starts' => sub {
+        # ensure the liveview-loading element is initially visible
+        my $loading_element = $driver->find_element('#liveview-loading');
+        ok($loading_element->is_displayed(), 'liveview-loading is initially visible');
+
+        # simulate the live view starting
```

Comment:
*that* is a good comment unlike the examples before as is provides new information and explains the not obvious next part, keep this